test/support/chr/list_pagination.rb

```diff
@@ -100,17 +104,18 @@ def chage_position_to_begin_and_check_pagination(factory_name, class_name, list_
       # Reload page
       visit('/admin')
       select_last_module_from_list(list_of_modules)
-      assert page.has_css?('div.item-title', count: @loaded_items)
+      assert page.has_css?('div.item-title', count: @loaded_items), "Number of elements on page don't mach number of elements what must be loaded: Present: #{page.all('div.item-title').count}, Must be: #{@loaded_items}"
       assert page.has_css?("a[data-id='#{@last_item.id}']")
       assert_not page.has_css?("a[data-id='#{@last_item_on_first_page.id}']")
       scroll_to_bottom
-      assert page.has_css?('div.item-title', count: class_name.count)
+      assert page.has_css?('div.item-title', count: class_name.count), "Number of elements on page don't match model.count: Present: #{page.all('div.item-title').count}, Must be: #{class_name.count}"
```

Comment:
Line is too long. [199/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.